doc/ref/peer.rst

```diff
@@ -99,6 +99,36 @@ to the manage and jobs runner functions.
 .. note::
     Functions are matched using regular expressions.
 
+.. _peer-wheel:
+
+Peer Wheel Communication
+=========================
+
+Configuration to allow minions to execute wheel from the master is done via
+the ``peer_wheel`` option on the master. The ``peer_wheel`` configuration follows
+the same logic as the ``peer`` option. The only difference is that access is
+granted to wheel modules.
+
+To open up access to all minions to all wheel:
+
+.. code-block:: yaml
+
+    peer_wheel:
+      .*:
+        - .*
+
+This configuration will allow minions with IDs ending in example.com access
+to the key wheel functions.
+
+.. code-block:: yaml
+
+    peer_wheel:
+      .*example.com:
+        - key.*
+
+.. note::
+    Functions are matched using regular expressions.
```

Comment:
I think you need a blank line below the `.. note::` for this to display as a proper "note" section.